symengine/number.h

```diff
@@ -185,6 +185,8 @@ class Evaluate
     virtual RCP<const Basic> exp(const Basic &) const = 0;
     virtual RCP<const Basic> erf(const Basic &) const = 0;
     virtual RCP<const Basic> erfc(const Basic &) const = 0;
+    virtual RCP<const Basic> erfinv(const Basic &) const = 0;
+    virtual RCP<const Basic> erfcinv(const Basic &) const = 0;
```

Comment:
Since all of these methods are not implemented, there's no real point having them here.